sdk/servicebus/service-bus/samples/javascript/receiveMessagesStreaming.js

```diff
@@ -46,8 +40,8 @@ async function main() {
       };
 
       const onErrorHandler = (err) => {
-        if (err.retryable === false) {
-          console.log("Receiver will be recreated. A fatal error occurred:", err);
+        if (err.retryable === true) {
+          console.log("Receiver will be recreated. A recoverable error occurred:", err);
           resolve();
         } else {
           console.log("Non-fatal error occurred: ", err);
```

Comment:
Now that we have moved away from using the "fatal" term in the `if` block, we should probably change the else block too.

Just `Error occurred` should suffice as we don't know if it is fatal or not.

We can do better in Track 2

Reply:
Sounds good. :)